src/ert/scheduler/scheduler.py

```diff
@@ -92,6 +85,9 @@ def __init__(
         ee_token: Optional[str] = None,
     ) -> None:
         self.driver = driver
+        self._ee_queue = ee_queue
```

Comment:
I am not overly happy with the `ee` shortcut. Should we just spell out `_ensemble_evaluator_queue instead?